frontend/src/features/contacts/contact/edit/Person/UpdatePersonForm.tsx

```diff
@@ -126,6 +125,17 @@ const UpdatePersonComponent: React.FC<FormikProps<IEditablePersonForm>> = ({
     }
   };
 
+  const isContactMethodInvalid = useMemo(() => {
```

Comment:
The update form is not displaying the warning when data is cleared or is etl'd with no phone/email/address.